index.html

```diff
@@ -655,27 +655,44 @@ <h4>Amanda Jepson</h4>
           </div>
 
         </div>
-          <div class="row"> 
+          <div class="row">
             <!--new change-->
             <!-- awesome code about me-->
             <!--charlesmotaroki@students.uonbi.ac.ke	-->
+            <div class="col-lg-3 col-md-6 d-flex align-items-stretch">
+              <div class="member" data-aos="fade-up" data-aos-delay="400">
+                <div class="member-img">
+                  <img src="assets/img/team/team-5.jpg" class="img-fluid" alt="">
+                  <div class="social">
+                    <a href=""><i class="bi bi-twitter"></i></a>
+                    <a href=""><i class="bi bi-facebook"></i></a>
+                    <a href=""><i class="bi bi-instagram"></i></a>
+                    <a href=""><i class="bi bi-linkedin"></i></a>

```

Comment:
You can add the LinkedIn links and remove the rest of the empty href tags

Reply:
Got it.